src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -92,7 +94,10 @@
     <ShouldGenerateNuSpec Condition="'$(PackagePlatforms)' == '' AND $(PackageTargetRuntime.Contains('-$(PackagePlatform)'))">true</ShouldGenerateNuSpec>
     <!-- build if PackagePlatforms is not specified and arch is x86 or AnyCPU -->
     <ShouldGenerateNuSpec Condition="'$(PackagePlatforms)' == '' AND ('$(PackagePlatform)' == 'x86' OR '$(PackagePlatform)' == 'AnyCPU')">true</ShouldGenerateNuSpec>
+    <!-- if we built a nuspec, also pack unless explicitly set -->
+    <ShouldCreatePackage Condition="'$(ShouldGenerateNuPkg)' == ''">$(ShouldGenerateNuSpec)</ShouldCreatePackage>
     <BuildDependsOn Condition="'$(ShouldGenerateNuSpec)' == 'true'">GenerateNuSpec</BuildDependsOn>
+    <BuildDependsOn Condition="'$(ShouldCreatePackage)' == 'true'">$(BuildDependsOn);CreatePackage</BuildDependsOn>
```

Comment:
We can support both options with the flags.  I felt that a default of building the nupkgs was what folks want from the open.  Also in the corefx build workflow we have more flexibility for where signing occurs so we need not separate nuspec creation and packing like we do in TFS.

Also the task is reusable to do a batch pack should we need to do so in the open.  It's the same one I'm using in TFS.